src/services/users.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const { User } = require('../utils/models/User');
+
+const getAllUsers = () => {
+  return User.findAll();
+};
+
+const addUser = (name) => {
+  return User.create({ name });
+};
+
+const getUserById = (userId) => {
+  return User.findByPk(userId);
+};
+
+const deleteUser = (userId) => {
+  return User.destroy({
+    where: {
+      id: userId,
+    },
+  });
+};
+
+const changeUser = async(userId, name) => {
+  const foundedUser = await User.findByPk(userId);
+
+  await foundedUser.update({ name });
+
+  return foundedUser.save();
+};
```

Comment:
so u find user in controller and here.
can we use just update and avoid 4 time call our database?
like we check if userId is correct === number, after we update direct by id(1 SQL call) and get updated or null